### PR TITLE
tagline and testimonial bugs (Admin and Client)

### PR DESCRIPTION
**Pull request by SevaDer14**
PT: https://www.pivotaltracker.com/story/show/179475229

### CODE REVIEW

File: src/views/InformationView.jsx [hunk 1]
```diff
@@ -37,8 +37,10 @@ const InformationView = () => {
       //setSections(response);
       setSections(information_view_sections.sections);
       let response = await Information.index();
-      setPinnedItems(response.pinned);
-      setOtherItems(response.other);
+      let pinnedItems = response.filter(item => item.pinned === true)
+      let otherItems = response.filter(item => item.pinned === false)
```
Comment (1CIM) [inline src/views/InformationView.jsx:41]: item.other? no?

**Review by 1CIM — COMMENTED** [review]
just a comment

File: src/views/InformationView.jsx (hunk 1 above)
Comment (SevaDer14) [reply src/views/InformationView.jsx:41]: no its correct

Comment (1CIM) [reply src/views/InformationView.jsx:41]: ah no i was wrong👍

**Review by 1CIM — APPROVED** [review]

**Pull request merged**